repository/copier.go

```diff
@@ -9,6 +9,20 @@ import (
 	"gopkg.in/src-d/go-billy.v3"
 )
 
+// Prepends the bucket dir name to the file name. The number of characters
```

Comment:
s/Prepends/addBucketName prepends/ following the standard way of writing docs in Go